GOSS_tests_for_sbps.md

```diff
@@ -50,3 +50,44 @@ Then run the below command:
     goss -g /opt/cray/tests/install/ncn/tests/goss-validate-iscsi-sbps-config.yaml validate
 
 This command will run the test on the individual node on which this was run where iSCSI is confgured.
+
+## Steps to view the test logs
+
+When the test is run on master/pit node (method #1), logs would be available at the below location on the system(s):
+On the master node, the high level tests results log will be available and on the individual worker node, we can see
+the detailed test results log.
+
+On the master node:
+
+    /opt/cray/tests/install/logs/print_goss_json_results/
+
+e.g.
```

Comment:
Put a blank line after the `e.g.` so that the next line will be formatted the same way as line 62.

Reply:
Done.